src/web/templates/user/screen/results.html

```diff
@@ -18,13 +18,13 @@ <h2 class="set-title text-center">{{ screen.name }}</h2>
                 <tbody>
                 {% for result in results %}
```

Comment:
```suggestion
                {% for board in result_boards %}
```
Would also need a rename